marimbabot_planning/CMakeLists.txt

```diff
@@ -29,3 +32,13 @@ target_link_libraries(lilypond_to_hitpoint ${catkin_LIBRARIES})
 install(TARGETS planning lilypond_to_hitpoint
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )
+
+# Action 
+add_action_files(
+  DIRECTORY src/actions
+  FILES hit_sequence.action
```

Comment:
[By convention](http://wiki.ros.org/ROS/Patterns/Conventions) message/service/action types should be `CamelCase`

Reply:
done